compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/diagnostic/BLangDiagnosticLocation.java

```diff
@@ -35,8 +35,9 @@
     private TextRange textRange;
 
     public BLangDiagnosticLocation(String filePath, int startLine, int endLine, int startColumn, int endColumn) {
-        this.lineRange = LineRange.from(filePath, LinePosition.from(startLine, 0), LinePosition.from(endLine, 0));
-        this.textRange = TextRange.from(startColumn, endColumn - startColumn);
+        this.lineRange = LineRange.from(filePath, LinePosition.from(startLine, startColumn),
+                LinePosition.from(endLine, endColumn));
+        this.textRange = TextRange.from(0, 0);
```

Comment:
Is it ok to hard code this? In the compiler we don't have this information. But the syntax errors may have this information right? Or is it using something else?

Reply:
It uses different `io.ballerinalang.compiler.syntax.tree.Node` to create the text range and parse it when creating `SyntaxDiagnostics`. hardcoding to "0" is done only for now. Will have to change the behaviour later.